pom.xml

```diff
@@ -515,6 +515,25 @@ THE SOFTWARE.
             </execution>
           </executions>
         </plugin>
+        <plugin>
+          <groupId>com.diffplug.spotless</groupId>
+          <artifactId>spotless-maven-plugin</artifactId>
+          <version>${spotless.version}</version>
+          <configuration>
+            <java>
+              <removeUnusedImports />
+            </java>
+          </configuration>
+          <executions>
+            <execution>
+              <!-- Runs in verify phase by default -->
+              <goals>
+                <!-- Can be disabled using -Dspotless.check.skip=true -->
```

Comment:
Document in https://github.com/jenkinsci/jenkins/blob/master/CONTRIBUTING.md ? Maybe we need and entry for the quickest-possible build there

Reply:
Done